internal/broker/plans.go

```diff
@@ -427,11 +427,15 @@ func empty() *map[string]interface{} {
 	return &empty
 }
 
-func createSchemaWithProperties(properties ProvisioningProperties, additionalParams, update bool, requiered []string) *map[string]interface{} {
+func createSchemaWithProperties(properties ProvisioningProperties, additionalParams, update bool, requiered []string, shootAndSeedSameRegion bool, shootAndSeedFeatureFlag bool) *map[string]interface{} {
 	if additionalParams {
 		properties.IncludeAdditional()
 	}
 
+	if shootAndSeedFeatureFlag && additionalParams && shootAndSeedSameRegion {
```

Comment:
Why you check additionalParams here?

Reply:
I treat this shootAndSeedSameRegion as additionalParam. Beside that, it is used to hide in endpoints and tests to hide additional parameter section depending on the context.

Reply:
So someone calling `createSchemaWithProperties` should set both `additionalParams` and `shootAndSameRegion` to true if he intends to enforce the same region.